internal/runbits/requirements/requirements.go

```diff
@@ -157,10 +157,16 @@ func (r *RequirementOperation) ExecuteRequirementOperation(requirementName, requ
 		requirementVersion = ""
 	}
 
+	origRequirementName := requirementName
 	if validatePkg {
 		pg = output.StartSpinner(out, locale.Tl("progress_search", "", requirementName), constants.TerminalAnimationInterval)
 
-		packages, err := model.SearchIngredientsStrict(ns, requirementName, false, false)
+		normalized, err := model.FetchNormalizedName(ns, requirementName)
+		if err != nil {
+			multilog.Error("Failed to normalize '%s': %v", requirementName, err)
+		}
+
+		packages, err := model.SearchIngredientsStrict(ns, normalized, false, false) // ideally case-sensitive would be true (PB-4371)
```

Comment:
I will create a JIRA ticket to address this once this PR is merged so I have a concrete link to this line.